packages/api/tests/lib/test-mmt.js

```diff
@@ -98,6 +99,26 @@ test.serial(
   }
 );
 
+test.serial.only(
```

Comment:
You probably want to remove `.only`

Reply:
That would be true.  And my git secrets only catches test.only(, maybe I should update that too.